content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -67,19 +75,29 @@ Next, create all the jobs with one kubectl command:
 
 ```shell
 kubectl create -f ./jobs
-job "process-item-apple" created
-job "process-item-banana" created
-job "process-item-cherry" created
+```
+
+The output is:
```

Comment:
nit: for consistency reason, would you agree is good to keep `similar to this:` in ?